venvs/common.py

```diff
@@ -8,9 +8,25 @@
 import attr
 import click
 import filesystems.native
+import pytoml
+import tomlkit
 import virtualenv as virtualenv_for_path
 
 
+def load_config(filesystem, locator):
+    with filesystem.open(locator.root.descendant("virtualenvs.toml")) as venvs:
+        text = venvs.read()
+        try:
+            return tomlkit.loads(text)
+        except ValueError:
+            return pytoml.loads(text)
+
+
+def dump_config(config, filesystem, locator):
+    with filesystem.open(locator.root.descendant("virtualenvs.toml"), "w") as venvs:
+        return venvs.write(tomlkit.dumps(config))
```

Comment:
Did you check to see if this blows up if you give it a normal dict? (Because `load_config` can return one, and something tells me that `tomlkit` today or in the future will say it takes `TOMLDocument`s, not just `dict`s).

Reply:
I can't remember why I had that logic with `pytoml`, so I removed it, which means `load_config` only returns `TOMLDocument`s now.